releasetool/__main__.py

```diff
@@ -165,8 +165,13 @@ def publish_reporter_start(github_token: str, pr: str):
 @click.option("--pr", envvar="AUTORELEASE_PR", default=None)
 @click.option("--status", type=bool, default=True)
 @click.option("--details", envvar="PUBLISH_DETAILS", default=None)
-def publish_reporter_finish(github_token: str, pr: str, status: bool, details: str):
-    releasetool.commands.publish_reporter.finish(github_token, pr, status, details)
+@_language_option()
```

Comment:
So I think all this decorator does is add a prompt for `--language` (see lines 101-107) which will need to be passed in somehow https://github.com/googleapis/releasetool/blob/08c518ed0212a385e7fbe0f945aa77db5352b820/releasetool/commands/publish_reporter.sh#L19-L24 Otherwise this PR looks good to me.

Reply:
I did a bit of manual testing, I think that it will default to the appropriate language:

https://github.com/googleapis/releasetool/blob/a6ed96ef7dd31fb1497f4ba3a68f9484f9ddf496/releasetool/__main__.py#L106

if it detects an appropriate file, would `autorelease` be running it in the appropriate context that we'd detect the `package.json`?